tests/sentry/integrations/slack/test_disable.py

```diff
@@ -57,13 +62,26 @@ def test_fatal_and_disable_integration(self):
             body=json.dumps(bodydict),
         )
         client = SlackClient(integration_id=self.integration.id)
-        with pytest.raises(ApiError):
+
+        with self.tasks() and pytest.raises(ApiError):
             client.post("/chat.postMessage", data=self.payload)
         buffer = IntegrationRequestBuffer(client._get_redis_key())
         assert buffer.is_integration_broken() is True
         integration = Integration.objects.get(id=self.integration.id)
         assert integration.status == ObjectStatus.DISABLED
 
+    @responses.activate
+    @with_feature("organizations:disable-on-broken")
+    def test_email(self):
+        client = SlackClient(integration_id=self.integration.id)
+        with self.tasks():
+            notify_disable(self.organization, self.integration, client._get_redis_key())
```

Comment:
This should pass `self.integration.provider.name` instead of the `Integration` object